plugins/modules/dconf.py

```diff
@@ -390,6 +407,64 @@ def reset(self, key):
         # Value was changed.
         return True
 
+    def load(self, key, path):
+        """
+        Load the config file in specified path.
+
+        if an error occurs, a call will be made to AnsibleModule.fail_json.
+
+        :param key: dconf directory for which the config should be set.
+        :type key: str
+
+        :param path: Remote configuration path to set for the specified dconf path.
+        :type value: str
+
+        :returns: bool -- True if a change was made, False if no change was required.
+        """
+        # Ensure key refers to a directory, as required by dconf
+        root_dir = key
+        if not root_dir.endswith('/'):
+            root_dir += '/'
+
+        # Read config to check if change is needed and passing to command line
+        try:
+            with open(path, 'r') as fd:
+                raw_config = fd.read()
+        except FileNotFoundError as ex:
+            self.module.fail_json(msg='dconf failed while reading configuration file with error: %s' % ex)
+
+        # Parse configuratoin file
+        config = ConfigParser()
+        try:
+            config.read_string(raw_config)
+        except Exception as e:
+            self.module.fail_json(msg='dconf failed while reading config with error: %s' % e)
+
+        # For each sub-directory, check if at least one change is needed
+        changed = any(
+            not self.variants_are_equal(self.read("%s%s/%s" % (root_dir, sub_dir, k)), v)
+            for sub_dir in config.sections()
+            for k, v in config[sub_dir].items()
+        )
+
+        if self.check_mode or not changed:
+            return changed
+
+        # Set-up command to run. Since DBus is needed for write operation, wrap
+        # dconf command dbus-launch.
+        command = [self.dconf_bin, 'load', root_dir]
+
+        # Run the command and fetch standard return code, stdout, and stderr.
+        dbus_wrapper = DBusWrapper(self.module)
+        rc, out, err = dbus_wrapper.run_command(command, data=raw_config)
```

Comment:
What happens to keys that already exist, but are not part of `raw_config`? Are these kept, or removed?

If the latter, the `changed` check above is not correct.

Reply:
If keys exist in the database but, are not part of `raw_config`  then they're kept. `changed` check should be correct!